src/configfile.ts

```diff
@@ -473,10 +480,12 @@ async function parseBenchmark(
         urlPath = url;
         queryString = '';
       }
-
       spec.url = {
         kind: 'local',
-        urlPath: await urlFromLocalPath(root, urlPath),
+        urlPath: await urlFromLocalPath(
+          root,
+          path.resolve(path.dirname(configFilePath), urlPath)
```

Comment:
Is this line right? Wouldn't this make the url path relative to the config file path, and not the root path specified in config?

The solution I thought made sense from the discussion in the issue was that the root is relative to the config path, but the url is then relative to the root.

Reply:
I changed my mind versus the discussion in the issue. I prefer everything being relative to the config file.

Reply:
May be worth updating the root documentation in the README.md for tachometer with this nuance.

Reply:
Done!

Reply:
Considering configs being alternatives to cli options, I see how urls being relative to the config file makes more sense.

This description in the README then sounds misleading though.
https://github.com/Polymer/tachometer/blob/7bf3c579d883c5f76803155780a639ad65df3cef/README.md?plain=1#L844

Reply:
The `--root` flag isn't used when there's a config file. In general `--config` excludes the other options.